web/taskmaster/src/app/components/registerform/registerform.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component } from '@angular/core';
+import { FormControl, FormGroup, FormGroupDirective, ReactiveFormsModule, Validators } from '@angular/forms';
+import { MatButtonModule } from '@angular/material/button';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { EMPTY, catchError } from 'rxjs';
+import { UserService } from '../../services/user.service';
+import { SnackBarService } from '../../services/snackBar.service';
+import { User } from '../../models/user.model';
+
+@Component({
+  selector: 'app-registerform',
+  standalone: true,
+  imports: [
+    ReactiveFormsModule,
+    MatInputModule,
+    MatFormFieldModule,
+    MatButtonModule,
+  ],
+  templateUrl: './registerform.component.html',
+  styleUrl: './registerform.component.scss'
+})
+export class RegisterformComponent {
+  disabled = false;
+
+  registerForm = new FormGroup({
+    name: new FormControl<string>('', Validators.required),
+    email: new FormControl<string>('', Validators.required),
+    password: new FormControl<string>('', Validators.required),
```

Comment:
It is recommended to have the confirm password. Uses a different way for the validator, I will make a PR for that.